staging/smoke-110.yaml

```diff
@@ -1,9 +1,9 @@
 smoke:
   schedule: "*/5 * * * *"
   config:
-    name: "upload-sync-110-5-30"
-    swarmReplicas: 5
-    target: "swarm"
+    name: "upload-sync-110-21-30"
+    swarmReplicas: 21
```

Comment:
We should probably take this from the `swarm` values, but I am not sure we can, since this is an independent helm chart.